app/scripts/modules/kubernetes/src/pipeline/stages/runJob/runJobStage.js

```diff
@@ -28,136 +29,30 @@ module.exports = angular.module('spinnaker.kubernetes.pipeline.stage.runJobStage
-      .then((namespaces) => {
-        this.namespaces = namespaces;
-      });
-
-    accountService.listAccounts('kubernetes')
-      .then((accounts) => {
-        this.accounts = accounts;
-      });
-
-
-    this.searchImages = (query) => {
-      kubernetesImageReader.findImages({
-        provider: 'dockerRegistry',
-        count: 50,
-        q: query
-      }).then((data) => {
-
-        if (this.pipeline.triggers) {
-          data = data.concat(this.pipeline.triggers.map((image) => {
-            image.fromTrigger = true;
-            return image;
-          }));
-        }
-
-        if (this.contextImages) {
-          data = data.concat(this.contextImages);
-        }
-
-        this.containers = _.map(data, (image) => {
-          return {
-            name: image.repository.replace(/_/g, '').replace(/[\/ ]/g, '-').toLowerCase(),
-            imageDescription: {
-              repository: image.repository,
-              tag: image.tag,
-              registry: image.registry,
-              fromContext: image.fromContext,
-              fromTrigger: image.fromTrigger,
-              fromFindImage: image.fromFindImage,
-              cluster: image.cluster,
-              account: image.account,
-              pattern: image.pattern,
-              stageId: image.stageId,
-              imageId: $filter('kubernetesImageId')(image),
-            },
-            imagePullPolicy: 'IFNOTPRESENT',
-            account: image.accountName,
-            requests: {
-              memory: null,
-              cpu: null,
-            },
-            limits: {
-              memory: null,
-              cpu: null,
-            },
-            ports: [{
-              name: 'http',
-              containerPort: 80,
-              protocol: 'TCP',
-              hostPort: null,
-              hostIp: null,
-            }],
-            livenessProbe: null,
-            readinessProbe: null,
-            envVars: [],
-            command: [],
-            args: [],
-            volumeMounts: [],
-          };
-        });
-
-      });
-    };
-
-    this.groupByRegistry = (container) => {
-      if (container.imageDescription) {
-        if (container.imageDescription.fromContext) {
-          return 'Find Image Result(s)';
-        } else if (container.imageDescription.fromTrigger) {
-          return 'Images from Trigger(s)';
-        } else {
-          return container.imageDescription.registry;
+    this.configureJob = () => {
+      return $uibModal.open({
+        templateUrl: require('./configureJob.html'),
+        controller: 'kubernetesConfigureJobController as ctrl',
+        size: 'lg',
+        resolve: {
+          stage: function() {
+            return angular.copy($scope.stage);
+          },
+          pipeline: function(){
+            return $scope.pipeline;
+          }
         }
-      }
+      }).result.then((stage) => {
+        $scope.stage = angular.copy(stage);
```

Comment:
@danielpeach I'm having trouble getting the stage to update here. I'm not sure what I'm doing wrong. When I update `$scope.stage`, the validation still fails and the updated pipeline JSON isn't there. Thoughts?